controllers/channels.js

```diff
@@ -175,7 +175,10 @@ module.exports = (app, passport, React, ReactDOMServer) => {
   app.post('/channels/import', async (req, res) => {
     const { data } = req.body;
```

Comment:
if you use `const { data, user } = req.body;` line 179 will look better with `user.accountData`

Reply:
agree, I have fixed that